examples/document-search/chroma_otel.py

```diff
@@ -0,0 +1,93 @@
+# /// script
```

Comment:
I think we should start adding more comments to the examples. It would be sufficient to write a little summary / docstring on module level explaining what we're doing in the script

Reply:
done